rest-service/manager_rest/test/endpoints/test_blueprints.py

```diff
@@ -130,8 +131,9 @@ def test_delete_blueprint(self):
                 post_blueprints_response['id'], 'blueprint.yaml'))
 
         # trying to delete a nonexistent blueprint
-        resp = self.delete('/blueprints/nonexistent-blueprint')
-        self.assertEquals(404, resp.status_code)
+        with self.assertRaises(exceptions.CloudifyClientError) as context:
```

Comment:
we can change that to a specific exception type (not the catch-all CloudifyClientError), and then we won't need to assert the 404